src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -374,13 +387,19 @@ export default {
 		gap: 8px;
 	}
 
-	& &__input {
+	&__input {
 		width: 300px;
 		height: var(--default-clickable-area);
-		:deep(input) {
-			height: var(--default-clickable-area) !important;
-			cursor: default;
-		}
+		margin: 0;
+		padding: 10px 6px 10px 12px;
+		border: 2px solid var(--color-border-maxcontrast);
+		border-radius: var(--border-radius-large);
+		background-color: var(--color-main-background);
+		font-size: var(--default-font-size);
+		text-overflow: ellipsis;
+		color: var(--color-main-text);
```

Comment:
not needed

Reply:
Only color or all of these styles? I tried to fake input appearance, to not import NcTextField component for that

Reply:
Only color